app/controllers/articles_controller.rb

```diff
@@ -20,6 +20,8 @@ def create
   end
 
   def show
+    @booking = Booking.new
+    @review = Review.new
```

Comment:
Sure. Since the booking form and review form are on the article page, we need to pass an empty instance to fill in the form. Happy to show how it works tomorrow